### ISSUE TITLE
Interval has negative coordinates

### ISSUE THREAD

**Issue by mjafin** [pcingola/SnpEff#71]
Hi Pablo,
I'm getting a weird error using Mfascicularis:
```
        NW_005093313.1  2       .       T       TTTCATAAACAAACAGATCTCCAGTCTCCCTGCAATGCAGGGCATACTTGGATGGTA       173.0   PASS     ADJAF=1;AF=1;BIAS=0:1;DP=29;HIAF=1.000;HICNT=29;HICOV=29;LSEQ=A;MQ=255;MSI=3;MSILEN=1;NM=2.0;ODDRATIO=0;PMEAN=20.2;PSTD=1;QSTD=1;QUAL=35.7;REFBIAS=0:0;RSEQ=TTCATACCTATGACATTTAC;SAMPLE=22Mmauritian;SBF=1;SHIFT3=0;SN=58;TYPE=Insertion;VARBIAS=29:0;VD=29       GT:DP:VD:AD:AF:RD:ALD   1/1:29:29:0,29:1:0,0:29,0
java.lang.RuntimeException: Interval has negative coordinates.
        Class        : Marker
        Start        : -54
        End          : 1
        ID           :
        Parent class : Chromosome
        Parent       : NW_005093313.1   0-3061 CHROMOSOME 'NW_005093313.1'
```
It's an insertion so I wonder why it's showing start as negative?

**Comment by pcingola** [comment]
Hi, 
  Could you send me the exact command line you 
used? (I'd like to reproduce the error condition in 
order to debug it). Also, are you using the latest 
version (4.1 B)?

**Comment by mjafin** [comment]
Hi Pablo,
Thanks for the prompt reply. The command I used was
``/apps/bcbio-nextgen/latest-devel/rhel6-x64/bin/snpEff -Xms750m -Xmx6g eff -dataDir /ngs/reference_data/genomes/Mfascicularis/Macaca_fascicularis_5/snpeff -formatEff -noLog -i vcf -o vcf Macaca_fascicularis /ngs/analysis/fma_RNASEQ/fma_RNASEQ/work/vardict/22Mmauritian.vcf.gz``

The version is ``SnpEff version SnpEff 4.1a (build 2015-01-14)`` (from bcbio-nextgen). Want me to try a more recent version?

**Comment by mjafin** [comment]
I went ahead and tried ``4.1b``, which did not produce the error. So something must have been fixed around coordinates in ``4.1b``?

**Comment by pcingola** [comment]
OK, so I guess I fixed the issue already. 
I'll close the bug report...
